WordPress/gutenberg
# Autocomplete: use KeyboardEvent.code instead of KeyboardEvent.keyCode

## What?


Refactor the `Autocomplete` component to rely on `code` instead of `keyCode` for keyboard events

## Why?


[`keyCode`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) is deprecated, and replaced by [`code`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code)

## How?


Easy swap of values

## Testing Instructions





The autocomplete component can be tested in the block editor in rich text blocks (e.g. autocompleting a user after typing `@`) — in particular, the arrow keys, the enter key, and the escape key

Make sure that there are no regressions compared to how the component behaves on `trunk`
